patches/linker/link_ntsc11.ld

```diff
@@ -104,3 +104,5 @@ ntsc11_EXIDeselect = 0x8135b298;
 ntsc11_EXISync = 0x8135ab5c;
 ntsc11_EXIImm = 0x8135a774;
 ntsc11_EXIImmEx = 0x8135a9d0;
+
+ntsc11_is_key_repeat = 0x81481460;
```

Comment:
It looks like other functions are calling a `get` wrapper for this global ala NTSC11 `0x81302de0`

But I think looking at the variable directly is fine

Reply:
Oh whoops, you're right, not sure how I missed that 🤦 Either approach is fine by me, though using the function would make it impossible to accidentally modify it. If we're not worried though I'll leave it as-is 🙂